testcase/smartcontract/testcase.go

```diff
@@ -17,17 +17,12 @@
  */
 package smartcontract
 
-import (
-	"github.com/ontio/ontology-test/testcase/smartcontract/api"
-	"github.com/ontio/ontology-test/testcase/smartcontract/native"
-	"github.com/ontio/ontology-test/testcase/smartcontract/neovm"
-	"github.com/ontio/ontology-test/testcase/smartcontract/wasmvm"
-)
+import "github.com/ontio/ontology-test/testcase/smartcontract/wasmvm"
 
 //Register test case
 func TestSmartContract() {
-	native.TestNative()
-	neovm.TestNeoVM()
+	//native.TestNative()
```

Comment:
Don't comment test case.
keep all of test cases can testable.

Reply:
modified the testcase